src/progressbar/test/progressbar.spec.js

```diff
@@ -81,6 +81,7 @@ describe('progressbar directive', function() {
     expect(bar.attr('aria-valuemax')).toBe('100');
     expect(bar.attr('aria-valuenow')).toBe('22');
     expect(bar.attr('aria-valuetext')).toBe('22%');
+    expect(bar.attr('aria-labelledby')).toBe('foo');
```

Comment:
do we want a test case for when the title is omitted?  that the title is `progressbar`.